src/System.Data.SqlClient/tests/ManualTests/SQL/InstanceNameTest/InstanceNameTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public static class InstanceNameTest
+    {
+        //[CheckConnStrSetupFact]
+        [Fact]
+        public static void ConnectToSQLWithInstanceNameTest()
+        {
+            string tcpConnStr = DataTestUtility.TcpConnStr;
+
+            string hostname = null;
+            string instanceName = null;
+            SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(tcpConnStr);
+            string[] dataSourceParts = null;
+            if ((dataSourceParts = builder.DataSource.Split('\\')).Length == 2)
+            {
+                hostname = dataSourceParts[0].Replace("tcp:", "");
+                instanceName = dataSourceParts[1];
+            }
+
+            if (hostname != null && instanceName != null && IsBrowserAlive(hostname))
```

Comment:
You could get rid of instanceName, since you're only using hostname and the whole connection string. You can also put the connection open into the Split's if statement, since that's the only time hostname gets defined anyway.